packages/extension/src/newtab/DndModal.tsx

```diff
@@ -105,23 +114,22 @@ const DoNotDisturbModal: React.FC<DoNotDisturbModalProps> = ({
       }}
     >
       <ModalCloseButton onClick={onRequestClose} />
-      <div className={styles.heading}>
-        <h3 className={styles.title}>Do Not Disturb</h3>
-        {isActive ? null : (
-          <p className={styles.description}>
-            Choose your preferences while you&apos;re on Do Not Disturb mode
-          </p>
-        )}
-      </div>
-      {renderForm()}
-      <div className={classnames(styles.footer, styles.centered)}>
-        <button
-          className={classnames(styles.done, styles.centered)}
-          onClick={(e) => (isActive ? setDndSettings(null) : handleSubmit(e))}
-        >
-          {isActive ? 'Turn off' : 'Done'}
-        </button>
+      <div className="px-16 text-center">
+        <h3 className="font-bold text-2xl text-white">Do Not Disturb</h3>
+        <p className="mt-1 text-sm text-salt-50">{getDescription()}</p>
```

Comment:
`text-sm` => `typo-subhead`
`text-salt-50` => `text-theme-label-secondary`

Reply:
I used `typo-callout` instead for a much accurate `0.9375` value based on the design spec.

Reply:
I will write these things down as I come across it, to help documenting in the future!